test/e2e/instrumentation/monitoring/custom_metrics_deployments.go

```diff
@@ -119,17 +119,28 @@ func StackdriverExporterPod(podName, namespace, podLabel, metricName string, met
 			},
 		},
 		Spec: corev1.PodSpec{
-			Containers: []corev1.Container{stackdriverExporterContainerSpec(StackdriverExporter, metricName, metricValue)},
+			Containers: []corev1.Container{stackdriverExporterContainerSpec(StackdriverExporter, namespace, metricName, metricValue)},
 		},
 	}
 }
 
-func stackdriverExporterContainerSpec(name string, metricName string, metricValue int64) corev1.Container {
+func stackdriverExporterContainerSpec(name string, namespace string, metricName string, metricValue int64) corev1.Container {
 	return corev1.Container{
 		Name:            name,
-		Image:           "k8s.gcr.io/sd-dummy-exporter:v0.1.0",
+		Image:           "k8s.gcr.io/sd-dummy-exporter:v0.2.0",
 		ImagePullPolicy: corev1.PullPolicy("Always"),
-		Command:         []string{"/sd_dummy_exporter", "--pod-id=$(POD_ID)", "--metric-name=" + metricName, fmt.Sprintf("--metric-value=%v", metricValue)},
+		Command: []string{
+			"/bin/sh",
+			"-c",
+			"./sd_dummy_exporter" +
+				" --pod-id=$(POD_ID)" +
+				" --pod-name=$(POD_NAME)" +
+				" --namespace=" + namespace +
+				" --metric-name=" + metricName +
+				fmt.Sprintf(" --metric-value=%v", metricValue) +
+				" --use-old-resource-model" +
+				" --use-new-resource-model",
```

Comment:
What is the meaning of passing both `--use-old-resource-model` and `--use-new-resource-model`? Will it export the same metric twice in different formats?

Reply:
Yes, with both flags it's exported in both formats.

Reply:
Ok, that makes sense.